src/model/AreaDataSource.ts

```diff
@@ -108,6 +108,12 @@ export default class AreaDataSource extends MongoDataSource<AreaType> {
           $set: {
             'climbs.gradeContext': '$gradeContext' // manually set area's grade context to climb
           }
+        },
+        {
+          $set: {
+            climbs: { $sortArray: { input: '$climbs', sortBy: { 'metadata.left_right_index': 1 } } },
+            children: { $sortArray: { input: '$children', sortBy: { 'metadata.leftRightIndex': 1 } } }
```

Comment:
can you confirm `children[]` contains actual area objects?  I think it the only contains a list of area _id's.  When we ask for an area.children[] in GQL, we're making another DB query here:
https://github.com/OpenBeta/openbeta-graphql/blob/develop/src/graphql/resolvers.ts#L203

Reply:
Oh yeah you're right! I've not tested this far because it throws a "No such operation as $sortArray" error.

Reply:
What if we did a $lookup for children above this `$set` so that we have the full object?

Reply:
> What if we did a $lookup for children above this $set so that we have the full object?

I thought the query would get too wildly complicated for a small performance gain, whereas the GQL field resolution way [in my comment above](https://github.com/OpenBeta/openbeta-graphql/pull/296#discussion_r1199792692) is more elegant and simpler (but requires an additional query).